.tool-versions

```diff
@@ -1,2 +1,2 @@
 nodejs 14.19.3
-python 3.11.1
\ No newline at end of file
+python 3.10.0
```

Comment:
Just double checking this change was intentional?

Reply:
Yes and no. For some reason, some dependency fails with 3.11. Not really related to this PR change, I might add it in a different PR to make it clear